<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWidget</class>
 <widget class="QWidget" name="ControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controls</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="0" column="2" rowspan="7">
    <widget class="QScrollBar" name="YScrollBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>-500</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEditX"/>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEditY"/>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEditZ"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="renderButton">
     <property name="text">
      <string>render</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="rendertime">
     <property name="text">
      <string>0ms</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDial" name="Zdial">
     <property name="maximum">
      <number>400</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QScrollBar" name="XScrollBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>-500</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEditX</tabstop>
  <tabstop>lineEditY</tabstop>
  <tabstop>lineEditZ</tabstop>
  <tabstop>renderButton</tabstop>
  <tabstop>Zdial</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
